### DISCUSSION TITLE
Looking for anserini-0.9.5-SNAPSHOT

### DISCUSSION THREAD

**Discussion by liv-daliberti** [castorini/anserini#2171] [Q&A]
Hello, I am looking for anserini-0.9.5-SNAPSHOT to use in a recompling against this code: 

https://github.com/webis-de/wi21-query-obfuscation-with-keyqueries

However, I see you don't have anserini-0.9.5 available anymore, and I was wondering why (and where I might be able to go to find that particular release).

Thank you!

**Comment by lintool** [comment]
You want the commit at https://github.com/castorini/anserini/blob/master/docs/release-notes/release-notes-v0.9.4.md - build at that point in the repo, and `anserini-0.9.5-SNAPSHOT-*` will be built in `target/`.

**Reply by liv-daliberti** [reply]
Got it! Many thanks! :)
